### PR TITLE
NO_JIRA: Adding high level documentation for ACS private APIs.

### PR DESCRIPTION
**Pull request by mpichura**

### CODE REVIEW

File: docs/acs-private-apis.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+                }
+            }
+        ]
+    }
+}
+```
+### Hx Insights Configuration API
+Get Hx Insights configuration.
+* Method: `GET`
+* Path: `config`
```
Comment (Pawel-608) [inline docs/acs-private-apis.md:182]:
```suggestion
* Path: `config/-default-`
```

Comment (mpichura) [reply docs/acs-private-apis.md]: Fixed

File: docs/acs-private-apis.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        * `knowledgeRetrievalUrl` [string]
+    * Sample
+```json
+{
+    "entry": {
+        "knowledgeRetrievalUrl": "http://dummy-host.xyz/knowledge-retrieval/bots"
+    }
+}
+```
+### Nodes API extension
```
Comment (krdabrowski) [inline docs/acs-private-apis.md:198]: API description is missing, like: "Get predicted values for node"

**Review by krdabrowski — APPROVED** [review]
Nice job - this was really missing!

**Review by azakrzewski-hy — APPROVED** [review]

File: docs/acs-private-apis.md (hunk 2 above)
Comment (mpichura) [reply docs/acs-private-apis.md:198]: Added

**Pull request merged**